sui/src/sui_commands.rs

```diff
@@ -420,6 +440,9 @@ pub async fn make_server(
     authority: &AuthorityPrivateInfo,
     committee: &Committee,
     buffer_size: usize,
+    consensus_committee: &ConsensusCommittee<Ed25519PublicKey>,
+    consensus_store_path: &PathBuf,
```

Comment:
It would be more idiomatic to have &Path

Reply:
fixed